### PR TITLE
Added GPU floor mod op

### LINKED ISSUE #52179: FloorMod on gpu
```
^f41959ccb2 (Manjunath Kudlur       2015-11-06 16:27:58 -0800 20) namespace tensorflow {
^f41959ccb2 (Manjunath Kudlur       2015-11-06 16:27:58 -0800 21) namespace functor {
bdae9c62caa (Andrew Selle           2016-10-18 16:08:51 -0800 22) // TODO(b/32239807) No GPU ops for mod yet.
^f41959ccb2 (Manjunath Kudlur       2015-11-06 16:27:58 -0800 23) }  // namespace functor
^f41959ccb2 (Manjunath Kudlur       2015-11-06 16:27:58 -0800 24) }  // namespace tensorflow
^f41959ccb2 (Manjunath Kudlur       2015-11-06 16:27:58 -0800 25)
```
TODO(b/32239807) on **2016-10-18**, and now **2021-9-23**
When will we develop this feature?

### PR DESCRIPTION
**Pull request by AdeshChoudhar**
Fixes #52179

### CODE REVIEW

File: tensorflow/core/kernels/cwise_op_floor_mod.cc [hunk 1]
```diff
@@ -22,6 +22,12 @@ REGISTER4(BinaryOp, CPU, "FloorMod", functor::floor_fmod, Eigen::half, bfloat16,
           float, double);
 
 #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+REGISTER4(BinaryOp, GPU, "FloorMod", functor::safe_floor_mod, uint8, uint16,
+          int16, int64_t);
+#if !defined(MLIR_GENERATED_GPU_KERNELS_ENABLED)
+REGISTER3(BinaryOp, GPU, "FloorMod", functor::floor_fmod, float, Eigen::half,
```
[the remark this answers is not in the document]
Comment (AdeshChoudhar) [reply tensorflow/core/kernels/cwise_op_floor_mod.cc:28]: @akuegel Thanks for clearing that out!

[the remark this answers is not in the document]
Comment (akuegel) [reply tensorflow/core/kernels/cwise_op_floor_mod.cc:28]: The JIT kernels for FloorMod are now enabled: e688b1e5dc87c67860bae7d477ae555ff5c9d0c9

[the remark this answers is not in the document]
Comment (AdeshChoudhar) [reply tensorflow/core/kernels/cwise_op_floor_mod.cc:28]: @akuegel Thanks for notifying me! Will you tell me is there anything more I can help with related to this issue?

[the remark this answers is not in the document]
Comment (akuegel) [reply tensorflow/core/kernels/cwise_op_floor_mod.cc:28]: I think right now not. We have hit some issues which caused this delay, but all known issues are resolved. But I will let you know if anything comes up.

[the remark this answers is not in the document]
Comment (AdeshChoudhar) [reply tensorflow/core/kernels/cwise_op_floor_mod.cc:28]: Alright! Thank you everyone! :handshake:

**Pull request closed without merging**